lib/ohai/common/dmi.rb

```diff
@@ -69,9 +69,9 @@ module DMI
         127 => "end_of_table_marker",
       }
 
-      # list of IDs to collect, otherwise we generate pages of hashes about cache chip size and whatnot
-      # See OHAI-260. When we can give the user a choice, this will be a default.
-      ID_TO_CAPTURE = [ 0, 1, 2, 3, 4, 6, 11 ]
+      # list of IDs to collect from config or default to a sane list that prunes
+      # away some of the less useful IDs
+      ID_TO_CAPTURE = Ohai.config[:plugin][:dmi][:ids] || [ 0, 1, 2, 3, 4, 6, 11 ]
```

Comment:
hm, i don't think we should force any user that wants to add a single DMI ID to have to specify the full list of defaults too.